cmd/influxd/launcher/query_test.go

```diff
@@ -3,8 +3,13 @@ package launcher_test
 import (
 	"bytes"
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
+	context2 "github.com/influxdata/influxdb/v2/context"
+	"github.com/influxdata/influxdb/v2/mock"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
@bnpfeife looks like the lint checks are not properly catching import grouping - we use `gci` for this in 1.x